roles/ceilometer-common/tasks/main.yml

```diff
@@ -0,0 +1,31 @@
+---
+- name: create ceilometer user
+  user: name=ceilometer comment=ceilometer shell=/bin/false system=yes home=/nonexistent
+        createhome=no
+
+- name: create ceilometer environment.d directory
+  file: dest=/etc/ceilometer/environment.d state=directory
+
+- name: create ceilometer log directory
+  file: dest=/var/log/ceilometer state=directory mode=0755 owner=ceilometer
+
+- name: create ceilometer cache dir
+  file: dest=/var/cache/ceilometer/api state=directory mode=0700
+        owner=ceilometer group=ceilometer
+
+- name: ceilometer config
+  template: src=etc/ceilometer/{{ item }} dest=/etc/ceilometer/{{ item }} mode=0644
+  with_items:
+    - ceilometer.conf
+    - policy.json
+    - api_paste.ini
+    - event_definitions.yaml
+    - event_pipeline.yaml
+    - pipeline.yaml
+
+- include: monitoring.yml tags=monitoring,common
+
+- include: logging.yml
+  tags:
+    - logrotate
+    - logging
```

Comment:
You use two different styles here for indicating tags on an include. Please pick one.